go.mod

```diff
@@ -10,11 +10,14 @@ require (
 	github.com/grpc-ecosystem/grpc-health-probe v0.3.6
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.3.0
 	github.com/mikefarah/yq/v3 v3.0.0-20201202084205-8846255d1c37
-	github.com/onsi/ginkgo v1.14.2
-	github.com/onsi/gomega v1.10.3
-	github.com/operator-framework/api v0.0.0-00010101000000-000000000000
-	github.com/operator-framework/operator-lifecycle-manager v0.0.0-00010101000000-000000000000
-	github.com/operator-framework/operator-registry v0.0.0-00010101000000-000000000000
+	github.com/mitchellh/hashstructure v1.0.0
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/onsi/ginkgo v1.16.1
+	github.com/onsi/gomega v1.11.0
+	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
+	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
+	github.com/operator-framework/api v0.9.2
+	github.com/operator-framework/operator-registry v1.13.6
```

Comment:
This looks off - cherrypick individual commits downstream to the staging directory shouldn't produce a diff here as we're always remapping these imports to the local filepath.

Reply:
Ah, I see what happened. Some of the changes here were introduced to the top-level pkg directory and not the staging path.